3rdParty/LightPcapNg/LightPcapNg/src/light_pcapng.c

```diff
@@ -442,6 +442,11 @@ char *light_pcapng_to_string(light_pcapng pcapng)
 
 uint32_t *light_pcapng_to_memory(const light_pcapng pcapng, size_t *size)
 {
+   if (pcapng == NULL) {
+		*size = 0;
```

Comment:
Possible null pointer dereference if `size = NULL`. Maybe we should add another `if` to check this

```
if (pcapng == NULL) {
    if (size != NULL) {
       *size = 0;
    }
   ....
}
```

Reply:
nice catch, fixed.

Reply:
I checked the code again looks like my second suggestion is better for remove the possibility completely because there are some other `*size =` calls in function. At first glance I misread the code sorry, even after this fix there is still possibility of null pointer dereference. So can you update the code with,

```
if (size == NULL) { // PCPP fix
   return NULL;
}
if (pcapng == NULL) {
  ...
}
```

Reply:
sure, seems better. just fixed.